dts/bindings/pwm/fake-pwm.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024 Kickmaker
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    This binding provides a fake PWM for use as either a stub or a mock in Zephyr
+    testing.
+
+compatible: "fake-pwm"
```

Comment:
prefix with `zephyr,`

Reply:
I updated the compatible and renamed the bindings